### DISCUSSION TITLE
A custom field for a checkbox list of images

### DISCUSSION THREAD

**Discussion by azim-kordpour** [filamentphp/filament#12560] [Help]
### Package

Form builder

### Package Version

v3

### How can we help you?

I want to create a custom checkbox list of images. It is displayed as I intend.
My problem is that I get null after submitting the form.
This is the view of my custom field:
```blade
@foreach($model->getMedia('slides') as $media)
    <div>
        <img src="{{$media->getFullUrl()}}">
        <input type="checkbox"
               value="{{$media->id}}"
               wire:model="{{ $getStatePath() }}.{{$loop->index}}"
        >
    </div>
@endforeach
```
How can I add a value to the array of this field after a checkbox is checked?

**Comment by xalabama** [comment]
Can you show up how you implemented that inside of your backend?
